software/SolarAISVN/SolarAISVN.ino

```diff
@@ -159,18 +162,14 @@ void measureVoltages() {
   // solar, battery, currentA, currentB, load, wind, dump, solar2, LiPo  
   voltage[0] = int((4096 - voltage[0]) * 7.52 - 1000);  // pin32 solar    voltage divider 10k : 1.2 k Ohm 1:1
   voltage[1] = int((4096 - voltage[1]) * 7.52 - 1000);  // pin33 battery  voltage divider 10k : 1.2 k Ohm 1:1
-  voltage[2] = int((voltage[2]) * 0.804 + 129);         // pin34 voltage solar minus green LED
-  voltage[3] = int((voltage[3]) * 0.804 + 129);         // pin35 voltage solar minus green LED minus 0.1 Ohm serial
+  //voltage[2] = int((voltage[2]) * 0.804 + 129);         // pin34 voltage solar minus green LED
+  //voltage[3] = int((voltage[3]) * 0.804 + 129);         // pin35 voltage solar minus green LED minus 0.1 Ohm serial
+  voltage[2] = int((voltage[3] - voltage[2]) * 6.75);   // voltage difference pin35 - pin34 x 8.4 is corrent (x0.804)
   voltage[4] = int((4096 - voltage[4]) * 7.52 - 1000);  // pin14 load     voltage divider 10k : 1.2 k Ohm 1:1
   voltage[5] = int((4096 - voltage[5]) * 7.52 - 1000);  // pin26 wind     voltage divider 10k : 1.2 k Ohm 1:1
   voltage[6] = int((voltage[6]) * 1.608 + 258);         // pin27 dump     not connected yet
   voltage[7] = int((voltage[7]) * 4.583 + 735);         // pin12 Solar2   voltage divider 4.7k : 1k
   voltage[8] = int((voltage[8]) * 1.608 + 258);         // pin13 LiPo     voltage divider 100kOhm 2:1
   Serial.print("Boot number: ");
   Serial.println(bootCount);  
-  Serial.print("Millisecond to measure: ");
-  //zeit = zeit - millis();
-  Serial.print(millis());
-  Serial.print(" - ");
-  Serial.println(zeit);
```

Comment:
Old code from 0.6 to display the submission time on the console in different WiFi situations, no longer needed.